docs/get-started/viam.md

```diff
@@ -15,143 +15,172 @@ imageAlt: "/general/understand.png"
 images: ["/general/understand.png"]
 ---
 
-Viam is a complete software platform that supports every step of your {{< glossary_tooltip term_id="smart-machine" text="smart machine">}} development lifecycle.
+Viam is a software platform that makes it easy to work with hardware and software.
+At the core of Viam is the open-source `viam-server` executable which:
 
-## Plan your machine
+- Creates, configures, and maintains any machine on which it is installed
```

Comment:
I fear that this section gets too technical and detailed too quickly.

However, who is our target audience for this particular bit?